src/query/service/src/servers/http/v1/query/http_query.rs

```diff
@@ -302,6 +302,8 @@ pub struct HttpSessionConf {
     /// last_query_ids[0] is the last query id, last_query_ids[1] is the second last query id, etc.
     #[serde(default)]
     pub last_query_ids: Vec<String>,
+    #[serde(default)]
+    pub last_query_result_cache_keys: Vec<String>,
```

Comment:
since client never need to aware it, I would prefer to put it in HttpSessionStateInternal